package/target_init.gradle

```diff
@@ -54,6 +54,7 @@ allprojects {
 	def buildDir = new File(System.getenv("runelite.pluginhub.package.buildDir"));
 
 	task runelitePluginHubPackage(type: Jar) {
+        duplicatesStrategy = 'warn'
```

Comment:
Why is this included?

Reply:
Basically when creating a fat jar it copies all the files into a single jar. Whenever there are duplicates like LICENSE or NOTICE it will break.

Now it will keep both duplicates and log a warning that it happened. This way there is no loss of files and the build will succeed.

Reply:
Why is this put here (which would affect all plugins) instead of in the build script for your plugin?

Reply:
ah thanks didn't know that was possible. fixed it in my branch, will clean up the git history when I'm home :)